### PR TITLE
[18.09] Minor UX cleanup for the check command

### PR DESCRIPTION
**Pull request by dhiltgen**
Clean up the naming a bit on the check command.

Signed-off-by: Daniel Hiltgen <daniel.hiltgen@docker.com>

/cc @mataneja

### CODE REVIEW

**Comment by dhiltgen** [comment]
```
 % ./docker engine --help

Usage:	docker engine COMMAND

Manage the docker engine

Commands:
  activate           Activate Enterprise Edition
  check-for-upgrades Check for available engine upgrades
  update             Update a local engine

Run 'docker engine COMMAND --help' for more information on a command.
```
```
% ./docker engine check-for-upgrades --help

Usage:	docker engine check-for-upgrades [OPTIONS]

Check for available engine upgrades

Options:
      --containerd string        override default location of containerd endpoint
      --downgrades               Report downgrades (default omits older versions)
      --format string            Pretty-print updates using a Go template
      --pre-releases             Include pre-release versions
  -q, --quiet                    Only display available versions
      --registry-prefix string   Override the existing location where engine images are pulled (default "docker.io/store/docker")
      --upgrades                 Report available minor or major upgrades (default true)
```

File: cli/command/engine/check.go [hunk 1]
```diff
@@ -28,8 +28,8 @@ func newCheckForUpdatesCommand(dockerCli command.Cli) *cobra.Command {
 	var options checkOptions
 
 	cmd := &cobra.Command{
-		Use:   "check [OPTIONS]",
-		Short: "Check for available engine updates",
+		Use:   "check-for-upgrades [OPTIONS]",
```
Comment (thaJeztah) [inline cli/command/engine/check.go]: Not a fan of naming it `check-for-upgrades`. Perhaps `list`, `browse`, `list-updates`? t.b.h. (but separate discussion), I think there's too many options on this command for filtering "types" of updates.

Wondering if we should consider adding a `--list` or  `--dry-run` option to `docker engine update`.

Comment (cpuguy83) [reply cli/command/engine/check.go]: `docker engine upgrades`?

**Comment by andrewhsu** [comment]
closing this PR because it doesn't look like it is desired anymore

**Pull request closed without merging**